mevislab.github.io/content/tutorials/summary/summary3.md

```diff
@@ -0,0 +1,691 @@
+
+Add the Field to the *Settings* UI and define *step = 0.1* and *slider = True*.
+
+Define a *Field* *isoValueImage* in *Parameters* section and set *internalName = IsoSurfaceImage.isoValue*,  *type = Integer*,  *min = 1* and  *max = 1000*.
+
+In *Settings* section of the UI, set *step = 2* and *slider = True*.
+
+{{< highlight filename="<MACRO_NAME>.script" >}}
+```Stan
+Interface {
+  Inputs {}
+  Outputs {}
+  Parameters {
+    Field openFile {
+      type         = String
+      internalName = LocalImage.name
+    }
+    Field selectOverlayColor {
+      internalName = SoView2DOverlay.baseColor
+      type         = Color
+    }
+    Field imageAlpha {
+      internalName = SoWEMRendererImage.faceAlphaValue
+      type         = Integer
+      min          = 0
+      max          = 1
+    }
+    Field thresholdInterval {
+      internalName = RegionGrowing.autoThresholdIntervalSizeInPercent
+      type         = Integer
+      min          = 0
+      max          = 100
+    }
+    Field isoValueImage  {
+      internalName = IsoSurfaceImage.isoValue
+      type         = Integer
+      min          = 0
+      max          = 1000
+    }
+  }
+}
+
+Commands {
+  source = $(LOCAL)/TutorialSummary.py
+}
+
+Window {
+  // Define minimum width and height
+  minimumWidth  = 400
+  minimumHeight = 300
+  // Vertical Layout and 4 Boxes with Horizontal Layout
+  Vertical {
+    Box Source {
+      layout = Horizontal
+      Field openFile {
+        browseButton = True
+        browseMode   = open
+      }
+    }
+    Box Viewing {
+      layout = Horizontal
+      Viewer View2D.self {
+        expandX = True
+        expandY = True
+        type    = SoRenderArea
+      }
+      Viewer SoExaminerViewer.self {
+        expandX = True
+        expandY = True
+        type    = SoExaminerViewer
+      }
+    }
+    Box Settings {
+      layout = Horizontal
+      Field selectOverlayColor {
+        title   = Color
+      }
+      Field imageAlpha {
+        step    = 0.1
+        slider  = True
+      }
+      Field thresholdInterval {
+        step    = 0.1
+        slider  = True
+      }
+      Field isoValueImage {
+        step    = 2
+        slider  = True
+      }
+    }
+    Box Info {
+      layout    = Horizontal
+    }
+  }
+}
+
+```
+{{</highlight>}}
+
+Your User Interface of the Macro Module should now look similar to this:
+
+![User Interface without Python Scripting](/images/tutorials/summary/Example3_4.png "User Interface without Python Scripting")
+
+For the next elements, we require Python scripting. Nevertheless, you are already able to use your application and perform the basic functionalities without writing any line of code.
+
+### Python scripting
+Python scripting is always necessary in case you do not want to re-use an existing Field for your User Interface but implement functions to define what happens in case of any event.
```

Comment:
an existing Field - should not be capitalized if not used as a name (in many places)